modules/ROOT/pages/index.adoc

```diff
@@ -4,13 +4,17 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: platform, arm, rest, soa, saas, api, proxy, design, develop, anypoint platform, studio, mule, devkit, studio, connectors, auth, exchange, api design, apikit, raml, application network, anypoint, arm, rest, soa, saas, api, proxy
 
-Use the Anypoint Platform to build an application network, a structured set of integrations between apps and services that are secure and easy to reuse.
+Use Anypoint Platform to build an application network, which is a structured network of applications, data, and devices connected by reusable APIs.
 
-== Explore the Platform with Exchange
+== Explore Anypoint Exchange
 
-https://www.anypoint.mulesoft.com/exchange/[Exchange] hosts elements created by MuleSoft and others that you can use to build your own integration apps (Mule apps) and APIs. The site includes connectors with examples and templates. Most connectors also have helpful links where you can explore reference guides and user guides. RAML fragments, custom packages, videos, and other items are available here.
+The APIs you build in Mulesoft to integrate applications and services are, by design, reusable and built with enterprise security in mind.
+You can discover these APIs, as well connectors (connections to external data sources), samples, and templates on 
```

Comment:
Re "connectors (connections to" --> "connectors (for connections to"? We usually think of connectors as things that provide connections to something, rather than the connections themselves.

Reply:
It seems obvious to me from context that there would be more involved than just the connector. 
Here's one of the sources for my choice: https://forums.mulesoft.com/questions/50625/difference-between-connector-and-endpoint.html

Reply:
but connectors are not connections

Reply:
I don't think we need to be this technical on this landing page. I'll revise to "...as well as connectors, samples, and templates..." and we can worry about definitions on the intro topic. Thanks for calling this out!

Reply:
Agree